src/main/scala/scala/tools/refactoring/implementations/oimports/ImportsOrganizer.scala

```diff
@@ -0,0 +1,79 @@
+package scala.tools.refactoring
+package implementations.oimports
+
+import scala.tools.nsc.Global
+
+abstract class ImportsOrganizers[G <: Global, U <: TreeToolbox[G]](val treeToolbox: U) {
```

Comment:
If this doesn't work, you could consider to nest all specific organizers (`DefImportsOrganizer` and `ClassDefImportsOrganizer`) into `ImportsOrganizers`, thereby 'saving' two classes.